packages/app-project/src/screens/ProjectHomePage/components/Hero/components/WorkflowSelector/components/WorkflowSelectButton/WorkflowSelectButton.js

```diff
@@ -8,29 +8,34 @@ import React from 'react'
 import theme from './theme'
 
 function WorkflowSelectButton (props) {
-  const { router, workflow } = props
-
-  const as = workflow.default
-    ? `${router.asPath}/classify`
-    : `${router.asPath}/classify/workflow/${workflow.id}`
-
-  const href = '/Classify'
-
-  return (
-    <Link as={as} href={href} passHref>
-      <Button label={workflow.displayName} />
-    </Link>
-  )
+  const { router, workflow, disabled, ...rest } = props
+
+  if (disabled) {
+    return (
+      <Button disabled={true} label={workflow.displayName} {...rest} />
```

Comment:
Sorry, I think I meant can this be a styled Box instead of a button?

Reply:
If we're using a Box, it probably makes more sense to put it in the `WorkflowSelector` component

Reply:
That's true, we might want to disable a workflow button for some other reason.